test/slack/test.tf

```diff
@@ -0,0 +1,12 @@
+# Simple test of the slack module, seeing if the result is what we expect.
+resource "aws_sns_topic" "slack_test" {
+  name = "Slack Test Topic"
+}
```

Comment:
Why not make the topic in the module as well?

Reply:
Only because I misread and thought it wasn't created in here. Fixed now.